ios/CovidShield/AppDelegate.m

```diff
@@ -124,3 +129,25 @@ - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 }
 
 @end
+#pragma mark Patch for BackgroundTask submission
+static IMP _BGTaskScheduler_submitTaskRequest_orig = NULL;
+
+static BOOL preSubmitTaskHook(id me, SEL selector, BGTaskRequest *req, NSError **perr) {
+  if([req isKindOfClass: [BGProcessingTaskRequest class]] &&
+     [req.identifier isEqualToString: @"app.covidshield.exposure-notification"]) {
```

Comment:
This is now the 4th time that "**app.covidshield.exposure-notification**" is in the codebase. It should be moved to a single spot. That may be tricky, considering it's in Typescript, Kotlin, Objective-C, and a info.plist file.